src/core/common/spaceallocator/spaceallocator.hpp

```diff
@@ -83,7 +83,6 @@ struct Partition;
  */
 struct OutdatedItem {
     StaticString<oci::cDigestLen>           mDigest;
-    size_t                                  mSize;
     SpaceAllocatorItf*                      mAllocator {};
     StaticFunction<cDefaultFunctionMaxSize> mFreeCallback;
```

Comment:
mFreeCallback more logically looks like propery of Partition. It could be passed to partition on Init through interface.